source/Octopus.Tentacle/Kubernetes/KubernetesConfig.cs

```diff
@@ -22,6 +22,8 @@ public static class KubernetesConfig
         public static string HelmChartVersionVariableName => $"{EnvVarPrefix}__HELMCHARTVERSION";
         public static string HelmChartVersion => GetRequiredEnvVar(HelmChartVersionVariableName, "Unable to determine Helm chart version.");
 
+        public static string BootstrapRunnerExecutablePath => GetRequiredEnvVar("BOOTSTRAP_RUNNER_EXECUTABLE_PATH", "Unable to determine Bootstrap Runner Executable Path");
```

Comment:
Happy to have the discussion but we've typically not had `_` in the names (see helm chart version above). So it'd be `BOOTSTRAPRUNNEREXEPATH`

Reply:
That's a fair point for consistency, but it does give me an aneurysm trying to read it with no separators... 😅

Reply:
Fixed 👍